javascript/javascript3/week3/art.js

```diff
@@ -0,0 +1,45 @@
+class Circle {
+    constructor(x, y, r, startAngle, endAngle, fillColor) {
+        this.x = x;
+        this.y = y;
+        this.r = r;
+        this.startAngle = startAngle;
+        this.endAngle = endAngle;
+        this.fillColor = fillColor;
+    }
+    draw(){
+        var canvas = document.querySelector("canvas");
+        var ctx = canvas.getContext("2d");
+        ctx.beginPath();
+        ctx.arc(this.x, this.y, this.r, this.startAngle, this.endAngle);
+        ctx.fillstyle = this.fillColor;
+        ctx.fill();
```

Comment:
To display your art in full screen, you must set the canvas width and height. 
You can do it by:

```
canvas.height = window.innerHeight;
canvas.width = window.innerWidth;
```